agent/uiserver/bindata_assetfs.go

```diff
@@ -15,7 +15,7 @@
 // pkg/web_ui/assets/codemirror/mode/ruby/ruby-ea43ca3a3bdd63a52811e8464d66134b.js
```

Comment:
You may need to also edit the CI configs in oss or ent, as this file is specially handled in merges.